#
# old_revision [40ea110392539e19c2d251569a3610c0051e0c64]
#
# patch "file1"
#  from [a43dc27b1c92cccc533ceb3a27035128e26e5b07]
#    to [400b49625ecf4bb7d2d7096ba86801bddb05bb9f]
#
============================================================
--- file1	a43dc27b1c92cccc533ceb3a27035128e26e5b07
+++ file1	400b49625ecf4bb7d2d7096ba86801bddb05bb9f
@@ -1,3 +1,4 @@ quux
 foo
 bar
 quux
+baz
\ No newline at end of file
